engine/common/rpc/convert/execution_data.go

```diff
@@ -0,0 +1,281 @@
+package convert
```

Comment:
this file was refactored from `convert.go`. I think you just need to apply the same fix to `ChunkExecutionDataToMessage` and can remove this file. The test should still work

Reply:
Addressed!